src/ontology/mondo.Makefile

```diff
@@ -266,79 +266,197 @@ $(TEMPLATES_DIR)/ROBOT_addMedGen_fromConflictResolution.tsv: tmp/July2023_CUIRep
+	$(MAKE) update-efo-subset -B
+	$(MAKE) update-clingen -B
+	$(MAKE) update-ordo-subsets -B
+	$(MAKE) update-nando -B
+	$(MAKE) update-medgen -B
+	$(MAKE)  subset-metrics -B && cp $(TMPDIR)/subset-metrics.tsv $(TMPDIR)/subset-metrics-after.tsv
+	@echo "Subset metrics before..."
+	cat $(TMPDIR)/subset-metrics-before.tsv
+	@echo "Subset metrics after..."
+	cat $(TMPDIR)/subset-metrics-after.tsv
+
+# This is the main pipeline to update all external content
+update-external-content-incl-rare:
+	$(MAKE) subset-metrics -B && cp $(TMPDIR)/subset-metrics.tsv $(TMPDIR)/subset-metrics-before.tsv
+	$(MAKE) update-efo-subset -B
+	$(MAKE) update-clingen -B
+	$(MAKE) update-ordo-subsets -B
+	$(MAKE) update-nando -B
+	$(MAKE) update-medgen -B
+	$(MAKE) update-orphanet-rare -B
+	$(MAKE) update-gard -B
+	$(MAKE) update-nord -B
+	$(MAKE) update-inferred-subset -B
+	$(MAKE) update-rare-subset -B
+	$(MAKE)  subset-metrics -B && cp $(TMPDIR)/subset-metrics.tsv $(TMPDIR)/subset-metrics-after.tsv
+	@echo "Subset metrics before..."
+	cat $(TMPDIR)/subset-metrics-before.tsv
+	@echo "Subset metrics after..."
+	cat $(TMPDIR)/subset-metrics-after.tsv
+
+######################################################
+##### Mondo Rare Disease Pipeline ####################
+######################################################
+
+##### Orphanet Rare ################
+
+$(TMPDIR)/orphanet-rare-subset.owl: $(SRC)
+	$(ROBOT) merge -i $(SRC) reason \
+		query --format ttl --query ../sparql/construct/construct-orphanet-rare-subset.sparql $@
+
+.PHONY: update-orphanet-rare
+update-orphanet-rare:
+	$(MAKE) $(TMPDIR)/orphanet-rare-subset.owl
+	grep -vE '^(subset: orphanet_rare)' $(SRC) > $(TMPDIR)/mondo-edit.tmp || true
+	mv $(TMPDIR)/mondo-edit.tmp mondo-edit.obo
+	$(ROBOT) merge -i $(SRC) -i $(TMPDIR)/orphanet-rare-subset.owl --collapse-import-closure false convert -f obo --check false -o $(SRC).obo
+	mv $(SRC).obo $(SRC) && make NORM && mv NORM $(SRC)
+
+##### GARD #########################
+
+# The complex part here is that we need to dynamically update the MONDO source code, i.e. 
+# MONDO:equivalentTo and MONDO:obsoleteEquivalentTo.
+
+.PHONY: update-gard
+update-gard:
+	$(MAKE) $(TMPDIR)/external/processed-gard.robot.owl
+	grep -vE '^(xref: GARD:|subset: gard_rare)' $(SRC) > $(TMPDIR)/mondo-edit.tmp || true
+	mv $(TMPDIR)/mondo-edit.tmp mondo-edit.obo
+	$(ROBOT) merge -i $(SRC) -i $(TMPDIR)/external/processed-gard.robot.owl --collapse-import-closure false \
+		query --update ../sparql/update/insert-gard-obsoletion-status.ru \
+		convert -f obo --check false -o $(SRC).obo
+	mv $(SRC).obo $(SRC) && make NORM && mv NORM $(SRC) && make deprecated_annotation_merging && make NORM && mv NORM $(SRC)
+
+##### NORD #########################
+
+.PHONY: update-nord
+update-nord:
+	make $(TMPDIR)/external/processed-nord.robot.owl -B
+	grep -vE '^(xref: NORD:|subset: nord_rare)' $(SRC) > $(TMPDIR)/mondo-edit.tmp || true
```

Comment:
Since NORD also includes xrefs, does this also need a query similar to `insert-gard-obsoletion-status.ru`?

Reply:
See slack, for now, I think its good like this!